src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -29,7 +29,8 @@
     private String message;
     private String url;
     private Long timeTaken;
-    private String id;
+    private String id;  // id can be in the form of <googleId><time> 
```

Comment:
A bit odd that we don't have a # between googleId and time. Also, give examples?